employees/common/exports.py

```diff
@@ -0,0 +1,140 @@
+
+
+def set_columns_width(worksheet, col_num, width_settings):
+    column_letter = get_column_letter(col_num)
+    column_dimensions = worksheet.column_dimensions[column_letter]
+    column_dimensions.width = width_settings[col_num - 1]
+
+
+def fill_headers_for_single_user(worksheet):
+    for col_num, column_title in enumerate(ExcelGeneratorSettingsConstants.HEADERS_FOR_SINGLE_USER.value, 1):
+        cell = worksheet.cell(row=ExcelGeneratorSettingsConstants.HEADERS_ROW.value, column=col_num)
+        cell.value = column_title
+        set_format_styles_for_main_cells(cell)
+        width_settings = ExcelGeneratorSettingsConstants.COLUMNS_WIDTH_FOR_SINGLE_USER.value
+        set_columns_width(worksheet, col_num, width_settings)
+
+
+def fill_data_in_reports_for_single_user(worksheet, reports):
+    row_number = ExcelGeneratorSettingsConstants.FIRST_ROW_FOR_DATA.value
+    for report in reports:
+        data_in_row = [
+            report.date,
+            report.project.name,
+            str(report.task_activities),
+            report.work_hours_str,
+            report.description,
+        ]
+        for col_num, cell_value in enumerate(data_in_row, 1):
+            cell = worksheet.cell(row=row_number, column=col_num)
+            if col_num == ExcelGeneratorSettingsConstants.DESCRIPTION_COLUMN_FOR_SINGLE_USER.value:
+                wrapped_alignment = Alignment(
+                    vertical=ExcelGeneratorSettingsConstants.VERCTICAL_TOP.value, wrap_text=True
+                )
+                cell.alignment = wrapped_alignment
+                cell.value = cell_value
+            elif col_num == ExcelGeneratorSettingsConstants.HOURS_COLUMN_FOR_SINGLE_USER.value:
+                cell.value = ExcelGeneratorSettingsConstants.TIMEVALUE_FORMULA.value.format(cell_value)
+                cell.number_format = ExcelGeneratorSettingsConstants.HOURS_FORMAT.value
+            else:
+                cell.value = cell_value
+        row_number += 1
+    return row_number
+
+
+def summarizing_reports(worksheet, last_row):
+    total_cell = worksheet.cell(row=last_row, column=ExcelGeneratorSettingsConstants.TOTAL_COLUMN.value)
+    total_cell.value = ExcelGeneratorSettingsConstants.TOTAL.value
+    set_format_styles_for_main_cells(total_cell)
+    total_hours_cell = worksheet.cell(
+        row=last_row, column=ExcelGeneratorSettingsConstants.HOURS_COLUMN_FOR_SINGLE_USER_REPORTS.value
+    )
+    total_hours_cell.value = ExcelGeneratorSettingsConstants.TOTAL_HOURS_FORMULA_FOR_SINGLE_USER.value.format(
+        last_row - 1
+    )
+    total_hours_cell.number_format = ExcelGeneratorSettingsConstants.TOTAL_HOURS_FORMAT.value
+    set_format_styles_for_main_cells(total_hours_cell)
+
+
+def generate_xlsx_for_single_user(author: CustomUser):
+    reports = author.get_reports_created()
+    workbook = Workbook()
+    worksheet = workbook.active
+    worksheet.title = f"{author.first_name} {author.last_name}"
```

Comment:
How about adding datetime of the report generation too?

Reply:
Adding datetime to each sheet? Or just to filename for instance:

    Reports single user - Jan Kowalski_Datetime-reports.xlsx
    Project reports - Sheet storm_datetime-reports.xlsx
    I think addding datetime to each sheet could be very ugly but this is just my opinion.

Reply:
I think adding to file name should be enough.